### ISSUE TITLE
Incorrect code marked correct 

### ISSUE THREAD

**Issue by AEA2002** [FreeCodeCamp/FreeCodeCamp#2548]
The following code was marked correct in  challenges/waypoint-create-bootstrap-wells
http://www.freecodecamp.com/challenges/waypoint-create-bootstrap-wells

    <div class="container-fluid">
       <h3 class="text-primary text-center">jQuery Playground</h3>
        <div class="row">
         <div class="col-xs-6"></div>
         <div class="col-xs-6"></div>
           <div class="well">

         <div class="col-xs-6">
           <div class="well"></div>
         </div>
         <div class="col-xs-6">
           <div class="well"></div>
        </div>
      </div>
     </div>
    </div>

**Comment by bugron** [comment]
Dear camper, your issue is related to:
https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2541
https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2417
https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2399
https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2398
https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2393

Please see [Help I've Found a Bug](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Help-I%27ve-Found-a-Bug).

**Comment by benschenker** [comment]
Hey @AEA2002 why shouldn't that be marked as correct?

**Comment by AEA2002** [comment]
@benschenker
There should only be two `class="col-xs-6"` divs
Also there  a extra div with `class="well"` (third line)
```
 <div class="col-xs-6"></div>
     <div class="col-xs-6"></div>
       <div class="well">

     <div class="col-xs-6">
       <div class="well"></div>
     </div>
     <div class="col-xs-6">
       <div class="well"></div>
    </div>
  </div>
```

**Comment by benschenker** [comment]
Closing as this has been merged in #3129
